src/lib/colors.js

```diff
@@ -85,39 +85,134 @@ const interpolateYesLimited = interpolateLab(
   colors.markers.background.yes
 );
 
-const definedAccessibilityColorScale = scaleLinear()
-  .domain([0, 0.2, 0.4, 0.6, 0.8, 1])
-  .range([
-    colors.markers.background.no,
-    colors.markers.background.limited,
-    interpolateYesLimited(0.25),
-    interpolateYesLimited(0.5),
-    interpolateYesLimited(0.75),
-    colors.markers.background.yes,
-  ]);
-
-export function interpolateWheelchairAccessibilityColors(propertiesArray: NodeProperties[]) {
+function calculateWheelchairAccessibility(propertiesArray: NodeProperties[]) {
   if (!propertiesArray || propertiesArray.length === 0) {
-    return colors.markers.background.unknown;
+    return { definedCount: 0 };
   }
-  const accessibilityValues = propertiesArray.map(isWheelchairAccessible);
+
+  // take a 'random' selection when there are too many places
+  const filterMod = Math.floor(propertiesArray.length / 30);
```

Comment:
-1: lodash has a nicer-looking `sample` method :)

I think it's a good idea, but out of curiosity, did this noticeably change speed?

Reply:
I didn’t measure really. But I was running with the react inspector plugin, this is pretty slow already. I can add some data on this.